src/components/ProgressCircleContainer.ts

```diff
@@ -67,7 +73,7 @@ export default class ProgressCircleContainer extends Component<ContainerProps, C
             animate: this.props.animate,
             circleThickness: this.props.circleThickness,
             className: this.props.class,
-            clickable: !!this.props.microflow || !!this.props.page,
+            clickable: !!this.props.microflow || !!this.props.page || !!this.props.nanoflow,
```

Comment:
`!!this.props.nanoflow,` will always be true, 
Should be `!!this.props.nanoflow.nanoflow,`